apollo-client/src/main/java/com/ctrip/framework/apollo/util/ConfigUtil.java

```diff
@@ -410,23 +412,38 @@ public boolean isPropertyNamesCacheEnabled() {
     return propertyNamesCacheEnabled;
   }
 
+  public boolean isPropertyFileCacheEnabled() {
+    return propertyFileCacheEnabled;
+  }
+
   private void initPropertyNamesCacheEnabled() {
-    String propertyName = ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE;
-    String propertyEnvName = ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE_ENVIRONMENT_VARIABLES;
+    propertyNamesCacheEnabled = getPropertyBoolean(ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE,
+            ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE_ENVIRONMENT_VARIABLES,
+            "false");
+  }
+
+  private void initPropertyFileCacheEnabled() {
+    propertyFileCacheEnabled = getPropertyBoolean(ApolloClientSystemConsts.APOLLO_PROPERTY_FILE_CACHE_ENABLE,
+            ApolloClientSystemConsts.APOLLO_PROPERTY_FILE_CACHE_ENABLE_ENVIRONMENT_VARIABLES,
+            "true");
+  }
+
+  private static boolean getPropertyBoolean(String propertyName, String envName, String defaultVal) {
```

Comment:
How about we keep it non-static?

```suggestion
  private boolean getPropertyBoolean(String propertyName, String envName, String defaultVal) {
```

Reply:
I rebase the code, fixed